cargo-shuttle/src/init.rs

```diff
@@ -271,26 +211,9 @@ fn copy_template(src: &Path, dest: &Path) -> Result<()> {
 }
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
-enum GitVendor {
-    GitHub,
-    BitBucket,
-    GitLab,
-}
-
-impl GitVendor {
-    fn as_str(&self) -> &'static str {
-        match self {
-            Self::GitHub => "https://github.com/",
-            Self::BitBucket => "https://bitbucket.org/",
-            Self::GitLab => "https://gitlab.com/",
-        }
-    }
-}
-
-impl std::fmt::Display for GitVendor {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        write!(f, "{}", self.as_str())
-    }
+enum GitDir {
```

Comment:
I think many of the derives are not needed

Reply:
`Copy` depends on `Clone`, and `Eq` also needs `PartialEq`. `Debug` is optional, but nice to have.

Reply:
Ah now I see where Copy is used. But Eq is not needed. Sorry for nitpicking.

Reply:
No worries : )
`Eq` has no methods, and it's only used to highlight that the comparisons are equivalence relations. The actual implementation is still in `PartialEq`. It's described here: https://doc.rust-lang.org/std/cmp/trait.Eq.html.